tdp/cli/commands/plan/test_reconfigure.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from click.testing import CliRunner
+from .reconfigure import reconfigure
+from ..init import init
+
+
+def test_tdp_plan_reconfigure(collection_path, database_dsn_path, vars, tmp_path):
```

Comment:
`tmp_path` is not used. Same for many tests.

Reply:
fixed